tests/code/test_events.py

```diff
@@ -29,4 +30,8 @@ def test_getEventsInvalidProgram():
     with pytest.raises(DoesNotExist):
         getEvents(program_id= "asdf")
 
+@pytest.mark.integration
+def test_currentUseris():
+    user = indicateInterestController()
```

Comment:
How does this work? I don't see that function name in your changes. And in any case, there's nothing here that is testing adding or deleting an insert.

Reply:
Don't know. We are working on adding new test cases. This might have already been resolved.